Documentation/mgcb.md

```diff
@@ -50,13 +50,16 @@ Set the target platform for this build. It must be a member of the TargetPlatfor
 * Xbox360
 * iOS
 * Android
-* Linux
+* DesktopGL
 * MacOSX
 * WindowsStoreApp
 * NativeClient
+* PlayStationMobile
```

Comment:
Blah github won't let me edit on the site and it's a pain to do it through sourcetree/command line. Mind accepting and I'll fix right away after with another PR?